### PR TITLE
Update version and fix install command

### PR DESCRIPTION
**Pull request by J-Thompson12**
Updated the version to 0.6.2 and fixed the command in install.

### CODE REVIEW

**Review by jpellizzari — APPROVED** [review]
Is this not already automated by the docs job?

File: docs/installation.md [hunk 1]
```diff
@@ -21,17 +21,17 @@ Weave GitOps currently supports SaaS versions of GitHub and GitLab.
...
 gitops version
 ```
 
 You should see:
 
 ```console
-Current Version: v0.6.1-10-g35945c8b
-GitCommit: 35945c8b
-BuildTime: 2022-01-25_22:13:54
+Current Version: v0.6.1-rc1-2-gcf7b11b8
```
Comment (yiannistri) [inline docs/installation.md:32]: Shouldn't this be `v0.6.2` ?

Comment (J-Thompson12) [reply docs/installation.md:32]: yeah sorry. I am used to this working automatically.

Comment (J-Thompson12) [reply docs/installation.md]: Fixed it.

**Pull request merged**